src/Type/Definition/Type.php

```diff
@@ -267,7 +267,7 @@ public static function assertType($type): Type
     /**
      * @api
      */
-    public static function getNullableType(Type $type): Type
+    public static function getNullableType(?Type $type): ?Type
```

Comment:
Rather than poisoning this method with nullability, I would rather clean up the problematic call site that does not check for null.

Reply:
I understand. I can clean up the problematic call site instead.

Reply:
I understand. I'll do that instead.

Reply:
Here's the new commit: https://github.com/webonyx/graphql-php/pull/895/commits/c8420c88cc0088755bf4f120f0d7e24df71eae52